aspnet/dnx/projects.rst

```diff
@@ -1,8 +1,652 @@
+Each of the values in the ``frameworks`` section of the *project.json* file is a framework that your project will compile. The above snippet will build for the full .NET Framework (dnx451) or Core CLR (dnxcore50). 
+
+If you pack this project, using ``dnu pack`` then you will get a NuGet package that looks like the following:
+
+.. image:: projects/_static/TargetPlatformNupkg.png
+
+Notice that the ``nupkg`` has a folder for each of the frameworks you specified, allowing this NuGet package to work on any of the frameworks. You can also specify this as follows:
+
+.. code-block:: json
+
+    {
+      "frameworks": {
+        "dnxcore50":{},
+        ".NETPortable,Version=v4.5,Profile=Profile7":{}
+      }
+    }
+
+With the above ``frameworks`` section, you'll generate a package with a *portable-net45+win* folder that will be used when running on platforms that match that portable profile.
+
+Adding dependencies
+-------------------
+
+You manage the dependencies of your application with the ``dependencies`` section of your *project.json* file. The dependencies are defined by name and version, where the runtime loaders determine what should be loaded.
+
+.. code-block:: json
+
+    {
+      "dependencies":{
+        "Microsoft.AspNet.Mvc": "6.0.0-beta4"
+      }
+    }
+
+The above *project.json* section tells the DNX that you depend upon ``Microsoft.AspNet.Mvc``, which means you also depend on everything that the MVC package depends on.
+
+The schema for dependency types adheres the following pattern:
+
+.. code-block:: json
+
+	{
+		"dependencies": {
+			"type": "object",
+			"additionalProperties": {
+				"type": [ "string", "object" ],
+				"properties": {
+					"version": {
+						"type": "string"
+					},
+					"type": {
+						"type": "string",
+						"default": "default",
+						"enum": [ "default", "build" ]
+					}
+				}
+			}
+		}
+	}
+		
+For additional information about how dependency versions are chosen, see `Dependency Resolution <https://github.com/aspnet/Home/wiki/Dependency-Resolution>`_.
+		
+Package dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+The DNX Utility (dnu) is responsible for all operations involving packages in your application. You use the **Install** command to download a package based on the package id and add it to your application::
+
+	dnu install <package id>
+	
+For a list of usage, arguments, and options of the ``install`` command, enter the following in the command windows::
+
+	dnu help install
+	
+For more information about ``dnu`` commands, see `DNX Utility <https://github.com/aspnet/Home/wiki/DNX-utility>`_.
+
+The more common way of installing packages is to just edit the *project.json* file. Editors like Visual Studio provide IntelliSense for all packages, making editing the file far easier than running ``dnu install``. However, you can use ``dnu install`` if you prefer. 
+
+Assembly references
+^^^^^^^^^^^^^^^^^^^
+
+You can also specify a list of framework assemblies for frameworks other than *dnxcore50*:
```

Comment:
"... for some frameworks <del>other than dnxcore50</del>:"

There are, and may be in the future, other frameworks for which framework assemblies are not relevant. The below description is probably good enough to explain when you use framework assemblies, but we can iterate on that if needed.